Document/0x08b-Reference-Apps.md

```diff
@@ -0,0 +1,71 @@
+# Reference applications
+
+The applications listed below can be used as training materials. Note: only the MSTG apps and Crackmes are tested and maintained by the MSTG project.
+
+## Android
+
+### Android Crackmes
```

Comment:
Could you please include one entry per Android crackme from https://github.com/OWASP/owasp-mstg/tree/master/Crackmes?